requests-for-comments/2023-10-31-onboarding-emails.md

```diff
@@ -0,0 +1,151 @@
+
+It's not the aim of this email to describe the content of every email, but rather to give a functional overview. Screenshots are provided for many, but not all steps. If you want to examine all content, Customer.io is the best place to do it. Ask Joe for access. 
+
+The onboarding email flow is managed by the marketing team, and is implemented with Customer.io - [which does have some issues](https://github.com/PostHog/meta/issues/104).
+
+## Onboarding for US and EU PostHog Cloud
+**Campaign in Customer.io:** Onboarding - All cloud users 
+
+The 'full' onboarding experience is delivered across some 'sub' campaigns, below, which trigger based on user properties. This is because we don't want to email users about how to use a feature until we've checked and taken action on whether they are subscribed to other tools.
+
+This campaign triggers when a user completes the `user signed up` event for the first time, on either PostHog US or PostHog EU, provided they also have a valid email address.
+
+1. Wait one hour.
+2. Send Welcome Email. 
+3. Check users' role in organization, using the `role_at_organization` property.
+    1. If `founder` then wait 12 hours, and send Joe's Newsletter Invite email.
+        1. Check if user clicked the subscription CTA.
+            1. If yes, add `newsletter_cta_clicked: true` to user.
+            2. If no, do nothing. 
+        2. Wait 12 hours
+    2. If `engineering` then wait 12 hours, and send Andy's Newsletter Invite email. 
+        1. Check if user clicked the subscription CTA.
+            1. If yes, add `newsletter_cta_clicked: true` to user.
+            2. If no, do nothing. 
+        2. Wait 12 hours
+    3. If `product` then wait 12 hours. 
+        1. Check if the user has performed `action created`
+            1. If yes, do nothing.
+            2. If no, send Advice for Product Teams email.
+        3. Wait 12 hours.
+    4. If `marketing` then wait 12 hours. 
+        1. Check if the user has performed `action created`
+            1. If yes, do nothing.
+            2. If no, send Advice for Marketing Teams email.
+        3. Wait 12 hours.
+    5. If `sales` then wait 12 hours. 
+        1. Check if the user has performed `action created`
+            1. If yes, do nothing.
+            2. If no, send Advice for Sales Teams email.
+        3. Wait 12 hours.
+    6. If not `founder` `engineering` `product` `marketing` or `sales` then wait 24 hours
+4. Wait until a weekday between 1PM and 5PM in the users' time zone (UTC fallback)
+5. Send AARRR intro article email
+6. Check if the user is in the `Subscribers to Session Replays` segment
+    1. If `Yes`, do nothing. (See Session replay onboarding flow)
+    2. If `No`, wait 1 day.
+        1. Wait until a weekday between 1PM and 5PM in the users' time zone (UTC fallback)
+        2. Assign to random cohort branch. 
+            1. 50% receive Session replay upsell email
+            2. 50% recieve Experiment: Personal Invite email
```

Comment:
This is part of an experiment which we're having to manually test due to C.io's quirks. It's currently too close to call.